### DISCUSSION TITLE
Barcode Scanner has trouble focusing on some phones

### DISCUSSION THREAD

**Discussion by Mag00n** [primefaces/community#2469] [PrimeFaces] [answered]
Hello, 

We are trying to implement the barcode scanner from PF Extensions 10. It works very well on my phone (newer iphone 14 pro). Some older phones wont focus on the barcode to allow the component to scan. Are there any hooks or functions I can tap into to help the phones focus? Any guidance or tips would be appreciated.

**Comment by melloware** [comment]
I assume you mean the QR Code component?  https://www.primefaces.org/showcase-ext/sections/qrCode/basicUsage.jsf

**Reply by Mag00n** [reply]
No, this one: https://www.primefaces.org/showcase-ext/views/codeScanner.jsf;jsessionid=node01wf0q73qxkbvglf19gi0rruuf45.node0

**Reply by melloware** [reply]
cc @jepsar any thoughts here?

**Reply by jepsar** [reply]
Not much we can do here other than seeing if there is an update of the third party library.

**Reply by jepsar** [reply, marked as answer]
https://github.com/primefaces-extensions/primefaces-extensions/issues/1652

**Comment by jepsar** [comment]
@melloware can we create categories here? There is no category for PrimeFaces Extensions yet. Should there be one? CC @onursenture

**Reply by melloware** [reply]
I think they have just lumped it all in PF.  I am ok with it as we don't get TOO many PFE requests here?

**Reply by jepsar** [reply]
Maybe a label in that case? Not sure if we ever want to filter those issues out.

**Reply by melloware** [reply]
Done!

**Reply by onursenture** [reply]
If needed, we can open a category, but I also think that a label under the PrimeFaces category will be sufficient.

**Reply by melloware** [reply]
@Mag00n this will be updated in PFE 14.0.5
